src/main/java/bio/terra/service/dataset/flight/ingest/IngestUtils.java

```diff
@@ -395,6 +401,16 @@ public static void deleteScratchFile(FlightContext context, GcsPdao gcsPdao) {
     gcsPdao.deleteFileByGspath(pathToIngestFile, bucketResource.projectIdForBucket());
   }
 
+  public static void deleteLandingFile(FlightContext context, GcsPdao gcsPdao) {
```

Comment:
It's not clear to me what a Landing File is, might be useful to define it here or use a different term. Maybe Staging File, or Scratch File?

Reply:
Yeah, I agree. It's worth a comment

Reply:
We already use Scratch in our codebase to denote temporary files. We should continue using it here I think.

Reply:
ok, fair, I'll rename the landing to scratch

Reply:
Actually, now I remember by I didn't...we have a bunch of methods that use scratch for file scratch already and there are gonna be collisions.  I'm gonna keep it as landing since in my mind that's easier that scratch type 1 vs scratch type 2

Reply:
Why not reuse the existing methods?

Reply:
Oh, because they're different file paths.  So they're both in the same scratch bucket but different blob ids within there

Reply:
Gah makes sense